Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+# What is an Inclusive Minute
+
+An inclusive minute (or inclusive moment) is an inclusivity practice designed for teams to easily add to existing meeting agendas.
+The practice is intentionally kept small, informal and uncontroversial to maximize the likelihood of adoption and uptake.
```

Comment:
I prefer the Oxford comma. That being said, can you guarantee that all topics will be noncontroversial? I think that's hard to promise.
```suggestion
The practice is intentionally kept small, informal, and uncontroversial to maximize the likelihood of adoption and uptake.
```